services/browsers/database.go

```diff
@@ -136,7 +136,7 @@ type Bookmark struct {
 	ImageURL  string `json:"imageUrl"`
 	Removed   bool   `json:"removed"`
 	Clock     uint64 `json:"-"` //used to sync
-	DeletedAt uint64 `json:"deletedAt"`
+	DeletedAt uint64 `json:"deletedAt,omitempty"`
```

Comment:
I made this `omitempty` as I don't expect `mobile` to use this property yet, so it should default to `0`.

This is required, because I need the `StoreBookmark` API of this service in tests to explicitly set a `DeletedAt` value for bookmarks. This requires `StoreBookmark` to honor that property when inserting entries into the DB.

Because `mobile` uses `StoreBookmark` from browser service, we need to ensure it doesn't break with this change (hence omitempty)